components/TitleBar.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+const Headline = styled.h1`
+  position: fixed;
```

Comment:
if you use position sticky, you don't need the top and left commands and not the awkward margin-top in your layout component.